### PR TITLE
Addresses Issue #99 and full implementation of Asana Calculate Cost Script

### PR DESCRIPTION
**Pull request by danielsclint**

### CODE REVIEW

**Comment by danielsclint** [comment]
Woo-hoo for repeatability!!!!

- I'll move an example to /scripts and keep some of the tests around in tests.

- I need to think through the ft object. In order to get rid of it, some of the code in calculate_cost will need to be refactored. I don't think this is a complicated undertaking, but it could be a little time consuming. I'm with you, though... I really didn't like creating the ft object to get everything to work.

- Got it on the pathing.

- I'll add some documentation, and maybe a little readme for usage.

- I have the kwargs for common flags that can radically change the results. I lost a couple of hours on tracking down that capacity_constraint flag. It was really hosing up my resultset. Of course, if I had docstrings, you would already know that...

I'll come back to this tomorrow.

**Comment by danielsclint** [comment]
The last few commits should address most of @e-lo comments. Please let me know if the docstrings are done correctly.

For ft object, I think we should put that in its own ticket.

**Comment by danielsclint** [comment]
Sorry, this got a little messy. I should have made the Travis changes on another branch.

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
 python:
   - "2.7"
+
 install:
-  - pip install -r requirements.txt
-  - pip install pytest
-# command to run tests
-script: pytest
\ No newline at end of file
+  - sudo apt-get update
```
Comment (e-lo) [inline .travis.yml]: @danielsclint  you couldn't help yourself :-)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/danielsclint/fast-trips.svg?branch=develop)](https://travis-ci.org/danielsclint/fast-trips)
```
Comment (e-lo) [inline README.md]: should refer to main repo

**Pull request merged**